impl/src/main/java/org/jboss/weld/logging/EventLogger.java

```diff
@@ -96,4 +96,7 @@
     @Message(id = 417, value = "The original observed type {0} is not assignable from {1} set by extension {2} - the observer method invocation may result in runtime exception!", format = Format.MESSAGE_FORMAT)
     void originalObservedTypeIsNotAssignableFrom(Object originalObservedType, Object observedType, Object extension);
 
+    @Message(id = 418, value = "Observer method for container lifecycle events cannot be static. {0}\n\tat {1}\n  StackTrace:", format = Format.MESSAGE_FORMAT)
```

Comment:
Hm, the same wording is used in `EventLogger.asyncContainerLifecycleEventObserver()`. We should fix the message there as well ;-)